Initial resultstoreui setup

craigericlewis opened the pull request:
Implement resultstoreui CLI functionality to get and create invocations as well as perform single uploads to invocations.

J-Pai commented on resultstoreui.py:
```diff
@@ -0,0 +1,510 @@
+    """
+    credentials, _ = auth.default(
+        scopes=("https://www.googleapis.com/auth/cloud-source-tools",
+                "https://www.googleapis.com/auth/cloud-platform"))
```
Make these scopes a global constant.

Combine with bigstore scopes below in a common credentials file.

craigericlewis commented:
> 1. Make a separate PR for generated python files and merge that in first.
Opened a new pr with just the generate python files here https://github.com/google/resultstoreui/pull/2

craigericlewis commented:
Made the request changes from @J-Pai.
1. Moved out resultstore api stuff into a separate client
2. Moved out bigstore uploader function into a separate client
3. Separate credential class
Other minor updates

zhm12359 commented:
I think you should use git commit --amend so the commit logs aren't contaminated. For now you should squash your commits

zhm12359 commented:
Also can you rebase? So that this PR won't show the python generated files.

J-Pai commented on resultstore_client.py:
```diff
@@ -0,0 +1,438 @@
+                                        config_id)
+        self.finalize_target(self.flags.invocation_id, target_id)
+
+    def file_upload_helper(self, target_id, action_id, config_id):
```
Is this meant to be publicly accessible? If not, note that it is private.

zhm12359 commented on resultstoreui.py:
```diff
@@ -0,0 +1,45 @@
+from resultstoreapi.cloud.devtools.resultstore_v2.proto import common_pb2
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('command', '', 'Command to execute')
```
list out possible values.
